app/forms/idv/doc_pii_form.rb

```diff
@@ -8,7 +8,8 @@ class DocPiiForm
     validate :dob_valid?
     validate :state_id_or_passport
 
-    attr_reader :first_name, :last_name, :dob, :state_id_type, :attention_with_barcode
+    attr_reader :first_name, :last_name, :dob, :address1, :state, :zipcode, :attention_with_barcode,
```

Comment:
we shouldn't need the state id fields since those were extracted out into DocPiiStateId

Reply:
Removed.